snuba/datasets/storages/events_common.py

```diff
@@ -233,7 +233,7 @@ def get_column_tag_map() -> Mapping[str, Mapping[str, str]]:
 def get_tag_column_map() -> Mapping[str, Mapping[str, str]]:
     # And a reverse map from the tags the client expects to the database columns
     return {
-        col: dict(map(reversed, trans.items()))
+        col: dict(map(reversed, trans.items()))  # type: ignore
```

Comment:
I think we can just ignore this, Mypy seems confused since the expected types of map (iterable) and reverse (sequence) don't match.